tests/unit/model/message_test.py

```diff
@@ -3122,6 +3124,7 @@ def test_merge_message_simple_derived() -> None:
                 Field("NR_F3"): deepcopy(ENUMERATION),
                 Field("NR_F4"): deepcopy(RANGE_INTEGER),
             },
+            byte_order=ByteOrder.HIGH_ORDER_FIRST,
```

Comment:
I think it would make sense to test if merging an inner message with a different byte order leads to the expected result.

Reply:
done